src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -419,8 +419,8 @@ export default class ConvasController {
         const angle = runway.angle;
 
         cc.translate(
-            round(window.uiController.km_to_px(runway.position[0])) + this.canvas.panX,
-            -round(window.uiController.km_to_px(runway.position[1])) + this.canvas.panY
+            round(window.uiController.km_to_px(runway.position.relativePosition[0])) + this.canvas.panX,
```

Comment:
Will be `runway.relativePosition[0]` or `runway.relativePosition[RELATIVE_POSITION_OFFSET_INDEX.LONGITUDINAL]`. I agree though, it would be nice to have `relativePosition` be an object so you can access the individual values by `.x` and `.y`.